app/controllers/hyrax/dashboard/works_controller.rb

```diff
@@ -19,6 +19,11 @@ def ensure_admin!
         def search_action_url(*args)
           hyrax.dashboard_works_url(*args)
         end
+
+        # The url of the "more" link for additional facet values
+        def search_facet_path(args = {})
+          hyrax.dashboard_works_facet_path(args[:id])
```

Comment:
Should this leverage `args.fetch(:id)`? It would fail with KeyError. I don't know how `dashboard_works_facet_path` handles a `nil` parameter.